src/main/java/mate/academy/Main.java

```diff
@@ -13,11 +12,9 @@ public static void main(String[] args) {
         manager.registerListener(sampleListener);
         manager.registerListener(loggingListener);
 
-        // Simulating an event
         Event event = new Event("UserLogin", "User123");
```

Comment:
The 'Event' class constructor is being used, but there is no import statement for it. Ensure that the 'Event' class is imported if it's not part of the same package.